src/bicep/modules/securityCenter.bicep

```diff
@@ -0,0 +1,105 @@
+targetScope = 'subscription'
+
+var bundle = (environment().name != 'AzureUSGovernment' ? [  
+  'KeyVaults'
+  'SqlServers'
+  'VirtualMachines'
+  'StorageAccounts'
+  'ContainerRegistry'
+  'KubernetesService'
+  'SqlServerVirtualMachines'
+  'AppServices'
+  'Dns'
+  'Arm'
+  ] : [
+  'SqlServers'
+  'VirtualMachines'
+  'StorageAccounts'
+  'ContainerRegistry'
+  'KubernetesService'
+  'Dns'
+  'Arm'
+])
+
+@description('Turn automatic deployment by ASC of the MMA (OMS VM extension) on or off')
+@allowed([
+  'On'
+  'Off'
+])
+param autoProvisioning string = 'On'
+
+@description('Turn security policy settings On or Off.')
+@allowed([
+  'On'
+  'Off'
+])
+param securitySettings string = 'On'
```

Comment:
elsewhere in the codebase and in this PR we use "enableX" 

Should we stick to that so that it's consistent and a boolean, then use a var to write the value these resources need (which is weird that the service doesn't accept them as booleans to begin with, but that's neither here nor there 😂)?

```bicep
param enableAutoProvisioning bool
var autoProvisioning = enableAutoProvisioning ? 'On' : 'Off'

param enableSecuritySettings bool
var securitySetting = enableSecuritySettings ? 'On' : 'Off'
```